xlsxwriter/workbook.py

```diff
@@ -683,6 +683,10 @@ def _check_sheetname(self, sheetname, is_chartsheet=False):
             else:
                 sheetname = self.sheet_name + str(self.sheetname_count)
 
+        # Check if sheetname is not empty
```

Comment:
Make the comment "Check if sheetname is empty.", with full stop.

Also make the conditional:

```python
if sheetname == '':
    ...
```